drivers/mysql/mysql.go

```diff
@@ -0,0 +1,395 @@
+// Initial code generated by generator.
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/go-morph/morph/drivers"
+	"github.com/go-morph/morph/models"
+	_ "github.com/go-sql-driver/mysql"
+)
+
+var driverName = "mysql"
+var defaultMigrationMaxSize = 10 * 1 << 20 // 10 MB
+var defaultConfig = &Config{
```

Comment:
nit: I think this is only used for getting default values, would it make sense to take an approach something like this:
```Go
func populateAndValidateConfig(cfg *Config) error {
	// define default config here
	// check fields and set default values
	// check for invalid values
}
```